src/main/scala/xiangshan/frontend/Tage.scala

```diff
@@ -230,13 +233,25 @@ class TageBTable(implicit p: Parameters) extends XSModule with TBTParams{
     ).reduce(_||_)
   )).asUInt
 
+  //Using WrBypass to store wdata dual ports for reading and writing to the same address.
+  val write_conflict = ((u_idx === s0_idx && io.update_mask.reduce(_||_)) || (doing_reset && resetRow === s0_idx)) && s0_fire
```

Comment:
doing_reset is power-on reset, I believe doing_reset and s0_fire are mutually exclusive.

Reply:
It will not be read during doing_reset, so there is no need to consider conflicts, right?